### PR TITLE
Add an invitation to contribute to TSPL below its docs links.

### PR DESCRIPTION
**Pull request by krilnon**
### Motivation:

Someone suggested that such an invitation in this part of the site might make it more obvious that we're open to contributions.

### Modifications:

Added a link and encouragement to contribute below the links to the Swift book on documentation/index.md.

### Result:

Readers are more aware that the book is open to contributions.

### CODE REVIEW

**Review by shahmishal — APPROVED** [review]

**Comment by shahmishal** [comment]
@swift-ci test

File: documentation/index.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ offering a guided tour, a comprehensive guide, and a formal reference of the lan
   </div>
 </div>
 
+We welcome your contributions to the book --- pull requests, new issues, and comments on existing topics --- in [the book's repository](https://github.com/apple/swift-book).
```
Comment (dempseyatgithub) [inline documentation/index.md:28]: I think the triple dashes are probably more correct as em-dashes, so:
"to the book—pull requests, new issues, and comments on existing topics—in”

https://www.grammarly.com/blog/why-you-should-love-the-em-dash/

Comment (dempseyatgithub) [inline documentation/index.md:28]: Smart quote: book’s

Comment (krilnon) [reply documentation/index.md:28]: I believe the Jekyll processing will result in an em dash.

Comment (krilnon) [reply documentation/index.md:28]: Same here; I think this is handled by the site generator.

[the remark this answers is not in the document]
Comment (krilnon) [reply documentation/index.md:28]: Here I followed precedent later down the page: "the [Swift.org repository](https://github.com/apple/swift-org-website)" doesn't use `swift-org-website` in the link text.

**Pull request merged**